web/src/scripts/controllers/inputs/edit-input.js

```diff
@@ -50,15 +50,12 @@
       vm.originalName = item.originalName;
 
       setTexts(item.texts);
+      vm.templateFragmentsData = fragmentTemplates;
+      vm.dataSource = angular.copy(item.fragmentSelected);
 
-      if (vm.policiesRunning.length === 0){
-        vm.templateFragmentsData = fragmentTemplates;
-        vm.dataSource = angular.copy(item.fragmentSelected);
-
-        vm.createTypeModels(vm.templateFragmentsData);
-        vm.selectedIndex = vm.index;
-        vm.policiesAffected = policiesAffected;
-      }
+      vm.createTypeModels(vm.templateFragmentsData);
+      vm.selectedIndex = vm.index;
+      vm.policiesAffected = policiesAffected;
     }
 
     function setPoliciesRunning(policiesList) {
```

Comment:
Could you delete this function and other text messages and variables related to policies running???